worker/Makefile

```diff
@@ -37,6 +37,13 @@ else
 	export PYTHONPATH := $(PIP_DIR):${PYTHONPATH}
 endif
 
+# Default Windows activate VS environment
```

Comment:
Dot please 😅

Reply:
I wish there was an automated check for this, I keep forgetting